libraries/lib-numeric-formats/formatters/ParsedNumericConverterFormatter.cpp

```diff
@@ -580,6 +581,403 @@ class ParsedNumericConverterFormatter final : public NumericConverterFormatter
+      XO("100>01000 octaves|1.442695041"),    // Scale factor is 1 / ln (2)
+      /* i18n-hint: an octave is a doubling of frequency */
+      XO("thousandths of octaves")
+   }
+   },
+
+   {
+   /* i18n-hint: Name of display format that shows log of frequency
+    * in semitones and cents */
+   { XO("semitones + cents") },
+   {
+      /* i18n-hint: Format string for displaying log of frequency in semitones
+       * and cents.
+       * Change the decimal points for your locale. Don't change the numbers.
+       * The decimal separator is specified using '<' if your language uses a ',' or
+       * to '>' if your language uses a '.'. */
+      XO("1000 semitones >0100 cents|17.312340491"),   // Scale factor is 12 / ln (2)
+      /* i18n-hint: a cent is a hundredth of a semitone (which is 1/12 octave) */
+      XO("hundredths of cents")
+   }
+   },
+   
+   {
+   /* i18n-hint: Name of display format that shows log of frequency
+    * in decades */
+   { XO("decades") },
+   {
+      /* i18n-hint: Format string for displaying log of frequency in decades.
+       * Change the decimal points for your locale. Don't change the numbers. */
+      XO("10>01000 decades|0.434294482"),   // Scale factor is 1 / ln (10)
+      /* i18n-hint: a decade is a tenfold increase of frequency */
+      XO("thousandths of decades")
+   }
+   },
+};
+
+Registry::BaseItemPtr MakeGroup (
+   const Identifier& identifier, NumericConverterType type, const BuiltinFormatString* formatStrings, const size_t count)
+{
+   auto group =
+      std::make_unique<NumericConverterRegistryGroup>(identifier, type);
+
+   for (size_t index = 0; index < count; ++index)
+   {
+      const auto functionIdentfier =
+         wxString::Format(L"%s_%zu", identifier.GET(), index);
+      
+      auto& formatString = formatStrings[index];
+
+      group->items.push_back(std::make_unique<NumericConverterRegistryItem>(
+         functionIdentfier, formatString.name,
+         formatString.formatStrings.fraction,
+         [&formatString, type](const auto& cfg)
+            -> std::unique_ptr<NumericConverterFormatter>
+         {
+            return std::make_unique<ParsedNumericConverterFormatter>(
+               type, formatString.formatStrings.formatStr.Translation(), cfg.sampleRate);
+         }));
+   }
+
+   return group;
+}
+
+NumericConverterItemRegistrator parsedTime {
+   Registry::Placement { {}, { Registry::OrderingHint::Begin } },
+   MakeGroup(
+      "parsedTime", NumericConverterType::TIME, TimeConverterFormats_,
+      WXSIZEOF(TimeConverterFormats_))
+};
+
+NumericConverterItemRegistrator parsedFrequency {
+   Registry::Placement { {}, { Registry::OrderingHint::Begin } },
+   MakeGroup(
+      "parsedFrequency", NumericConverterType::FREQUENCY,
+      FrequencyConverterFormats_, WXSIZEOF(FrequencyConverterFormats_))
+};
+
+NumericConverterItemRegistrator parsedBandwith {
+   Registry::Placement { {}, { Registry::OrderingHint::Begin } },
```

Comment:
The three "begin" hints can't all be satisfied -- but Registry will sort by the internal identifiers for determinacy.

But I think the ordering of this registry really doesn't matter for anything the user sees.